src/components/RangeSlider/types.ts

```diff
@@ -0,0 +1,43 @@
+import {Action} from '../Labelled';
+
+import {Error} from '../../types';
+
+export type DualValue = [number, number];
+export type RangeSliderValue = number | DualValue;
+
+export interface Props {
+  /** Label for the range input */
+  label: string;
+  /** Adds an action to the label */
+  labelAction?: Action;
+  /** Visually hide the label */
+  labelHidden?: boolean;
+  /** ID for range input */
+  id?: string;
+  /** Initial value for range input */
+  value: RangeSliderValue;
+  /** Minimum possible value for range input */
+  min?: number;
+  /** Maximum possible value for range input */
+  max?: number;
+  /** Increment value for range input changes */
+  step?: number;
+  /** Provide a tooltip while sliding, indicating the current value */
+  output?: boolean;
```

Comment:
I had to look this up because of the <output> html tag.  Would this be more clear as `tooltip?: boolean`

Reply:
That would be a breaking change, unfortunately, because this prop already exists in the single range slider.